website/docs/reference/dependent-features.md

```diff
@@ -46,11 +46,11 @@ To make use of dependent feature, you need to use a compatible client. Client SD
 
 - [unleash-client-node](https://github.com/Unleash/unleash-client-node) (from v4.2.0)
 - [unleash-client-java](https://github.com/Unleash/unleash-client-java) (from v8.4.0)
-- [unleash-client-go](https://github.com/Unleash/unleash-client-go) (from v3.9.0)
+- [unleash-client-go](https://github.com/Unleash/unleash-client-go) (from v3.9.2)
```

Comment:
We also have 4.0.1 because of the hashing problem, right?

Reply:
but dependent features were added before that

Reply:
Oh! I see now! This is just dependent features only. :+1: